src/config/external-applications.md

```diff
@@ -0,0 +1,51 @@
+# External Applications
+
+## Programming Languages
+
+> The Void repositories have a number of python and lua packages. If possible,
+> try to install packages from the Void repos using xbps-install or use xbps-src
+> to package your python or lua application. Packaging your application with
+> xbps-src will keep your system clean. If you send your package upstream it
+> will allow other Void users to benifit and allow them to help you maintain
+> your package. To learn how to package for xbps-src see the
+> [manual](https://github.com/void-linux/void-packages/blob/master/Manual.md)
+> and [contributing
+> instructions](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md).
```

Comment:
To start off, use Python, Lua and Linux when referring to them, unless you're talking about the package itself.

I'm having a complicated thought process regarding this one. People who develop in those languages sometimes don't really have the option of depending on distro packages, due to versioning issues and a bunch of other stuff. Perhaps something closer to:

```suggestion
> The Void repositories have a number of Python and Lua packages. If possible,
> install packages from the Void repositories or consider packaging the library or application you need.
> Packaging your application allows for easier system maintenance and can benefit
> other Void Linux users, so consider making a pull request for it. The contribution instructions can be found
>  [here](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md).
```